sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -228,12 +229,17 @@ func TestClientCertificateCredential_Live(t *testing.T) {
 }
 
 func TestClientCertificateCredentialADFS_Live(t *testing.T) {
-	certData, err := os.ReadFile(liveSP.pfxPath)
+	if recording.GetRecordMode() == recording.LiveMode {
+		if adfsLiveSP.certPass == "" || adfsLiveSP.clientID == "" || adfsLiveSP.pfxPath == "" || adfsLiveSP.scope == "" || adfsLiveSP.tenantID == "" {
+			t.Skip("this test requires manual recording and access to ADFS instance, and can't pass live in CI")
+		}
+	}
```

Comment:
A few comments here:
1. Live and recording mode are distinct but have the same configuration requirements i.e. the distinction is between playback mode and everything else
1. The skip message could point to what this test needs for a live run
1. Won't the tenant always be "adfs"?
```suggestion
	if recording.GetRecordMode() != recording.PlaybackMode {
		if adfsLiveSP.certPass == "" || adfsLiveSP.clientID == "" || adfsLiveSP.pfxPath == "" || adfsLiveSP.scope == "" {
			t.Skip("set ADFS_SP_* environment variables to run this test live")
		}
	}
```